packages/valory/skills/abstract_round_abci/tests/test_abci_app_chain.py

```diff
@@ -365,6 +366,11 @@ def end_block(self) -> None:
             (self.app1_class, sync_data_cls_app1),
             (self.app2_class, sync_data_cls_app2),
         ):
+            with caplog.at_level(logging.WARNING):
+                synchronized_data = sync_data_cls(db=AbciAppDB(setup_data={}))
+                abci_app = abci_app_cls(synchronized_data, MagicMock(), logging)  # type: ignore
+                expected = f"No `synchronized_data_class` set on {abci_app}"
+                assert expected in caplog.text
```

Comment:
an additional test for the log message when `synchronized_data_class` has not been set.